src/resources/your-label/section-one/view.html

```diff
@@ -515,11 +515,11 @@ <h2>Food with extra requirements</h2>
     </li>
   </ul>
 </section>
-<section>
+<!--<section>
```

Comment:
Hey @asifamingov , instead of commenting it, can we delete this part?

Reply:
Hi @nishgakkhar  this is only a temp solution.